Src/MvcPages/Configuration/TelluriumConfigurationSection.cs

```diff
@@ -21,7 +21,7 @@ public BrowserType Browser
             set { this[BrowserKey] = value;  }
         }
 
-        [ConfigurationProperty(DriversKey, DefaultValue = "Drivers", IsRequired = false)]
+        [ConfigurationProperty(DriversKey, DefaultValue = "", IsRequired = false)]
```

Comment:
Please restore default value as it was before.  The reason is simple: when you add drivers directly to your project and use "copy to output" option VS keep the path in output directory (subfolder is copied as well). Keeping drivers in the main project's directory can cause a mess and confusion. When default value is ""Drivers" a options in config file driversPath="" should override default value correctly.